CHANGELOG.md

```diff
@@ -8,6 +8,8 @@ Currently, this repo is in Prerelease. When it is released, this project will ad
 
 ## Prerelease
 
+- Changes `Brand` and `Utility` colors to match Figma main file
```

Comment:
The removal of `ui-success` and `ui-error` should be highlighted under a "Breaking Changes" section so that everyone who's upgrading knows to make those changes.

Reply:
Agree with @hokei here.

Reply:
OK, I made some changes to the wording in the CHNAGELOG.

Reply:
>  I searched the DS codebase and noticed that var is not used in any other files

Just to clarify in case people are searching in the future -- consuming apps can use these CSS variables directly, so any changes here should come with instructions on what to use instead.

Reply:
@hokei 
Ah, yes, in that light, this makes much more sense.  Thanks for the catch andn suggestion.